resteasy-jaxrs/src/main/java/org/jboss/resteasy/spi/ResteasyDeployment.java

```diff
@@ -475,6 +475,7 @@ public void registration()
       {
          registry.addPerRequestResource(actualResourceClass);
       }
+      registry.checkAmbiguousUri();
```

Comment:
@rsearls, is there a reason you put the test before processing the ResourceFactories?